src/chainparams.cpp

```diff
@@ -436,11 +437,33 @@ class CRegTestParams : public CChainParams
         pchMessageStart[2] = 0xb5;
         pchMessageStart[3] = 0xda;
         nDefaultPort = 18444;
-        nPruneAfterHeight = args.GetBoolArg("-fastprune", false) ? 100 : 1000;
+        nPruneAfterHeight = opts.fastprune ? 100 : 1000;
         m_assumed_blockchain_size = 0;
         m_assumed_chain_state_size = 0;
 
-        UpdateActivationParametersFromArgs(args);
+        for (const auto& [name, height] : opts.activation_heights) {
```

Comment:
`name` is really `deployment_id` (`dep` for short)?

Reply:
I see, I'll use `dep`, because that's what is used elsewhere in the code.